setup.cfg

```diff
@@ -149,3 +130,42 @@ multi_line_output = 0
 balanced_wrapping = True
 include_trailing_comma = false
 length_sort_stdlib = true
+
+[coverage:run]
```

Comment:
Ah, interesting. So you got rid of `coveragerc` file altogether.

Reply:
Indeed! Thanks @Cadair for letting me know this is possible :)